server/opts.go

```diff
@@ -1678,11 +1691,54 @@ func parseExportStreamOrService(v interface{}, errors, warnings *[]error) (*expo
 				*errors = append(*errors, err)
 			}
 		}
-
 	}
 	return curStream, curService, nil
 }
 
+// parseServiceLatency returns a latency config block.
+func parseServiceLatency(v interface{}) (serviceLatency, error) {
+	latency, ok := v.(map[string]interface{})
+	if !ok {
+		return serviceLatency{}, fmt.Errorf("Expected latency entry to be a map/struct, got %T", v)
+	}
+
+	var sl serviceLatency
+
+	if v, ok := latency["sample"]; ok {
+		tok, ok := v.(token)
+		if !ok {
+			return serviceLatency{}, fmt.Errorf("Expected latency token, got %T", tok)
+		}
+
+		sampleStr, ok := tok.Value().(string)
```

Comment:
This should already be parsed as a number, not a string.

Reply:
We can support "100%" as well, but if I just do `sample: 50` we should parse that correctly.

Reply:
Just so I understand, do we want to support
```
latency: {
  sample: 100   # <--- int
  subject: "latency.tracking.add"
}
```
and
```
latency: {
  sample: 100%  # <--- string
  subject: "latency.tracking.add"
}
```
¿?
Or just the int form?

Reply:
I like to be flexible in the config. I also think we support just `latency: "latency.metrics"`.

I think sample -> sampling.